src/test/java/edu/hm/hafner/analysis/parser/PVSStudioParserTest.java

```diff
@@ -0,0 +1,63 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.util.Locale;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertj.SoftAssertions;
+
+import edu.hm.hafner.analysis.parser.pvsstudio.PVSStudioParser;
+
+/**
+ * Tests the class {@link PVSStudioParser}.
+ *
+ * @author PVS-Studio Team
+ */
+class PVSStudioParserTest extends AbstractParserTest {
+    private static final String PREFIX = "pvsstudio/";
+
+    PVSStudioParserTest() {
+        super(PREFIX + "TestReport.plog");
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(33);
+        softly.assertThat(report.getFiles()).hasSize(1);
+
+        softly.assertThat(report).hasSeverities(1, 5, 24, 3);
+
+        softly.assertThat(report.filter(Issue.byType("General Analysis")).getSize()).isEqualTo(7);
```

Comment:
Can you extract these variables in `AnalyzerType` and reuse these constants here? Otherwise we need to change the name in two places.